docs/reST/ref/image.rst

```diff
@@ -202,10 +215,23 @@ following formats.
 
    .. note:: :func:`fromstring` and :func:`tostring` are named with "string,"
              but they deal in bytes. In Python 2, str = bytes, so this is
-             "from string" as in a Python 2 string.
+             "from string" as in a Python 2 string. The use of :func:`frombytes`/
+             :func:`tobytes` is preferred over :func:`fromstring`/:func:`tostring`
+             as of pygame 2.1.3.
 
    .. ## pygame.image.fromstring ##
 
+.. function:: frombytes
+
+   | :sl:`create new Surface from a byte buffer`
+   | :sg:`frombytes(bytes, size, format, flipped=False) -> Surface`
+
+   added :func:`formbytes` as an alias for :func:`formstring`,
+   the use of :func:`formbytes` is recommended over :func:`formstring`.
+   .. versionadded:: 2.1.3 
```

Comment:
There needs to be a newline between this sentence and the version added.

The generated checks don't come out properly otherwise, I checked.